doc/src/main/asciidoc/inc/_enricher.adoc

```diff
@@ -1017,6 +1020,63 @@ The trigger is added to all containers that apply, but can be restricted to a li
 ...
 ----
 
+[[fmp-configmap-file]]
+==== fmp-configmap-file
+
+This enricher adds ConfigMap defined as `resources` in plugin configuration and/or resolves file content from an annotation.
+
+As XML you can define:
+
+[source, xml]
+.pom.xml
+----
+<configuration>
+  <resources>
+    <configMap>
+      <elements>
+        <element>
+          <name>A</name>
+          <value>B</value>
+        </element>
+       </elements>
+    </configMap>
+  </resources>
+</configuration>
+----
+
+This creates a ConfigMap data with key `A` and value `B`.
+
+You can also use `file` tag to refer to the content of a file.
+
+[source, xml]
+----
+<configuration>
+  <resources>
+    <configMap>
+      <elements>
```

Comment:
what about using `<entries>` and `<entry>` instead of `<element>` ? That would match nicely with the "map" semantics which also refers to entries.